### ISSUE TITLE
(Bug report) Trilium fails to start after hard stop, hangs with full CPU utilization

### ISSUE THREAD

**Issue by johndoe0815** [zadam/trilium#3501]
### Trilium Version

0.57.4

### What operating system are you using?

Other Linux

### What is your setup?

Server access only

### Operating System Version

Linux trilium 5.15.83-1-pve #1 SMP PVE 5.15.83-1 (2022-12-15T00:00Z) x86_64 GNU/Linux

### Description

I have trilium running in a proxmox container. Everything was working well, until the host had a power failure. After restart, all my containers started up again properly, except trilium.
The node process is continuously utilizing the CPU and trilium is not accessible anymore. Is there any way to fix my installation or do I need to recover the last backup (with which I would lose a few late changes)?

**Comment by sigaloid** [comment]
SQLite is supposed to be resistant to power failure, any ongoing transactions will be rolled back upon opening it the next time. Is there any log messages? Anything about performing a recovery especially with the -wal files? (These are crucial to recovery in the case of a power outage)

**Comment by johndoe0815** [comment]
No, trilium was just not reachable over its web interface by IP:8080 and CPU remained fully utilized.
But as I needed my notes, I managed to download the last automatic weekly database backup and rolled back a previous complete backup with that. However with that experience I will now create even more frequent backups.

**Comment by zadam** [comment]
Hi, it would help if you could provide log files from the container when you were experiencing the problem.s

**Comment by eliandoran** [comment]
Closing as stale.
